chat-android/src/main/java/com/ably/chat/Messages.kt

```diff
@@ -70,6 +70,41 @@ interface Messages : EmitsDiscontinuities {
      */
     suspend fun send(text: String, metadata: MessageMetadata? = null, headers: MessageHeaders? = null): Message
 
+    /**
+     * Update a message in the chat room.
+     *
+     * This method uses the Ably Chat API REST endpoint for updating messages.
+     * It creates a new message with the same serial and a new version.
+     * The original message is not modified.
+     * Spec: CHA-M8
+     *
+     * @param messageCopy The updated copy of the message created using the `message.copy` method.
+     * @param operationDescription Optional description for the update action.
+     * @param operationMetadata Optional metadata for the update action.
+     * @returns updated message.
+     */
+    suspend fun update(messageCopy: Message, operationDescription: String? = null, operationMetadata: OperationMetadata? = null): Message
```

Comment:
I would use `message`, similar to delete, `messageCopy` sounds odd, at least `updated` or `updatedMessage`

Reply:
I would go with `updatedMessage` since normal `message` param doesn't convey type of message that needs to be sent.

Reply:
https://github.com/ably/ably-chat-kotlin/pull/100/commits/99a0fa27c48be1e7a56e1ea846ec6927a4d8a6cf